lib/settings_base.py

```diff
@@ -1008,6 +1008,11 @@ def JINJA_CONFIG():
     'addons.tasks.unindex_addons': {'queue': 'priority'},
     'addons.tasks.save_theme': {'queue': 'priority'},
     'addons.tasks.save_theme_reupload': {'queue': 'priority'},
+
+    # These are basically devhub.
+    'api.tasks.process_webhook': {'queue': 'devhub'},
+    'api.tasks.process_webhook': {'queue': 'devhub'},
```

Comment:
It is.

On Tue, Jan 19, 2016 at 2:01 PM, Mark Striemer <notifications@github.com>
wrote:

> In lib/settings_base.py
> <https://github.com/mozilla/olympia/pull/1145#discussion_r50182844>:
>
> > @@ -1008,6 +1008,11 @@ def JINJA_CONFIG():
> >      'addons.tasks.unindex_addons': {'queue': 'priority'},
> >      'addons.tasks.save_theme': {'queue': 'priority'},
> >      'addons.tasks.save_theme_reupload': {'queue': 'priority'},
> > +
> > +    # These are basically devhub.
> > +    'api.tasks.process_webhook': {'queue': 'devhub'},
> > +    'api.tasks.process_webhook': {'queue': 'devhub'},
>
> Is one of these supposed to be api.tasks.process_results?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/olympia/pull/1145/files#r50182844>.
>